### PR TITLE
feat(follow-up): enable zeebe contextPath for management port

### PR DESCRIPTION
**Pull request by aabouzaid**
### Which problem does the PR fix?

Fixes: https://github.com/camunda/camunda-platform-helm/issues/1392

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [x] In the repo's root dir, run `make go.update-golden-only`.
- [x] There is no other open [pull request](../pulls) for the same update/change.
- [x] Tests for charts are added (if needed).
- [x] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [x] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by aabouzaid** [comment]
@npepinpe For some reason, with or without `MANAGEMENT_SERVER_BASEPATH` var, when this endpoint is called `/zeebe/v1/topology` it returns this error:
```
{"type":"about:blank","title":"Not Found","status":404,"instance":"/zeebe/v1/topology"}
```

**Comment by npepinpe** [comment]
So MANAGEMENT_SERVER_BASEPATH is only for things served over 9600, i.e. the monitoring port. SPRING_WEBFLUX_BASEPATH is the one for the context path on port 8080.

Yesterday I tested with SNAPSHOT and the context path worked for the REST API (i.e. port 8080) 🤔

**Comment by aabouzaid** [comment]
@npepinpe it's working as expected when I used `SPRING_WEBFLUX_BASEPATH` but not clear to me now if we need `SERVER_SERVLET_CONTEXT_PATH` :thinking:

File: charts/camunda-platform/templates/zeebe-gateway/deployment.yaml [hunk 1]
```diff
@@ -76,7 +76,11 @@ spec:
             - name: ZEEBE_GATEWAY_MONITORING_PORT
               value: {{ .Values.zeebeGateway.service.httpPort | quote }}
             {{- if .Values.zeebeGateway.contextPath }}
-            - name: SPRING_WEBFLUX_BASE_PATH
+            - name: SERVER_SERVLET_CONTEXT_PATH
```
Comment (npepinpe) [inline charts/camunda-platform/templates/zeebe-gateway/deployment.yaml:79]: :x: Should be removed, this does nothing currently for Zeebe.

**Review by npepinpe — APPROVED** [review]

**Comment by npepinpe** [comment]
Quick question, what's the use case here, especially when reusing the same context path for both servers? Is it so one can server both the management and main server over the same port/DNS? Is this a common thing?

**Comment by aabouzaid** [comment]
> Quick question, what's the use case here, especially when reusing the same context path for both servers? Is it so one can server both the management and main server over the same port/DNS? Is this a common thing?

Curretly we just set them the same so they are accessible externally, so could be moninotred from external source.
But in the feature we need to standardized that a bit for all components with different path and potentially special auth.

**Pull request merged**